bookserver/models.py

```diff
@@ -559,6 +559,19 @@ class UserChapterProgres(Base, IdMixin):
     status = Column(Integer, nullable=False)
 
 
+class UserState(Base, IdMixin):
+    __tablename__ = "user_state"
+
+    user_id = Column(Integer, index=True)
```

Comment:
Is this a foreign key in some other table? If so, it should be marked that way.

All these should have NOT NULL constraints.